en/docs/ipaas/micro-integrator/develop-integrations-with-integration-studio.md

```diff
@@ -55,6 +55,42 @@ Conducting security vulnerability scans is crucial for identifying potential wea
 
 During deployment, Choreo performs a Trivy scan to detect critical vulnerabilities in third-party libraries added to the integration component. If the scan detects any critical vulnerabilities, it halts the deployment process. The deployment pane shows the Trivy scan status and any security failures in the Library (Trivy) vulnerable scan step, which you can access by clicking on it. After fixing the vulnerability, you can redeploy the component.
 
+## Configuring Micro Integrator
+
+By default, WSO2 MI comes with a set of pre-configured settings that are designed to work in a general context. However,
+these default configurations may not be suitable for every organization's specific needs. Therefore, it is important to
+change the default configuration in WSO2 MI to optimize its performance and ensure its compatibility with the
+organization's systems and applications.
+
+To modify the default configurations in WSO2 MI, users can apply their desired configurations in the instances that runs
+on the Choreo. To add new configurations, users need to define a `deployment.toml` file in the project sub-path.
+Similar to configuring MI locally, users can apply configuration changes in the `deployment.toml`. However, it's
+essential to apply only the required changes to avoid breaking the internal communication. It's worth noting that
+changing default configurations like port offset, hostname, and other critical settings might break the internal
+communication. Therefore, it's advisable to avoid modifying them unless it's necessary.
```

Comment:
```suggestion
To customize the default configurations in WSO2 MI, users can apply their desired configurations to the instances running on Choreo. To add new configurations, users need to define a `deployment.toml` file in the project sub-path. Similar to configuring MI locally, users can apply configuration changes in the `deployment.toml` file. However, it is essential to apply only the required changes to avoid disrupting internal communication. It's worth noting that altering default configurations such as port offset, hostname, and other critical settings might break internal communication. Therefore, it's advisable to avoid modifying them unless necessary.
```